docs/devsetup/source/index.md

```diff
@@ -81,7 +81,7 @@ contribute to ICU.)
 *   Install a **git client**
     *   <https://git-scm.com/downloads>
     *   Linux: `sudo apt install git`
-*   Install **git-lfs** if your git client does not already have LFS support
+*   For ICU 76 and previous versions: install **git-lfs** if your git client does not already have LFS support
```

Comment:
74 and previous

After that we didn't have jar files (after the move to maven)

Reply:
I'll keep it at 76 based on the work and fix version for [ICU-22967](https://unicode-org.atlassian.net/browse/ICU-22967), and that guidance is guaranteed to not be wrong for users.

Reply:
[ICU-22967](https://unicode-org.atlassian.net/browse/ICU-22967) only removed `old-xmls.tar.bz2`, which is not used (and was damaged already for a long-long time)
It was not even a bz2 anymore, it was a lfs link text.

So 74 is also guaranteed to not be wrong :-)

But your call.